File: src/com/company/Residential.java
package com.company;

import java.util.Scanner;

public class Residential extends Customer {
    public static void getInfo() {
        Scanner scan = new Scanner(System.in);

        System.out.print("Customer name: ");
        customerName = scan.nextLine();

        System.out.print("Customer phone number: ");
        customerPhone = scan.nextLine();

        System.out.print("Customer address: ");
        customerAddress = scan.nextLine();

        System.out.print("Square footage: ");
        squareFootage = scan.nextInt();

        System.out.print("Senior discount? (true or false): ");
        boolean senior = scan.nextBoolean();
        double weeklyRate = 6; // per 1000 feet

        double totalWeeklyCharges = (weeklyRate * squareFootage);

        if(senior){
            System.out.println("\n\nYou are receiving a senior discount.");
            System.out.println(customerName);
            System.out.println(customerAddress);
            System.out.println(customerPhone);
            System.out.println(squareFootage);
            System.out.println("Your charges are $" + String.format("%.2f", totalWeeklyCharges * .85));
            System.out.print("\n");

        }
        else {
            System.out.println("You are receiving no discount.");
            System.out.println(customerName);
            System.out.println(customerAddress);
            System.out.println(customerPhone);
            System.out.println(squareFootage);
            System.out.println("Your charges are $" + String.format("%.2f", totalWeeklyCharges));
            System.out.print("\n");

        }

    }
}

File: src/Residential.kt
class Residential(customerName: String,
                  customerPhone: String,
                  customerAddress: String,
                  squareFootage: Double) :
    Customer(customerName, customerPhone, customerAddress, squareFootage) {

    var isSenior : Boolean = false
    var rate : Int = 6

    fun billing() {
        val total = if (isSenior) {
            (rate * squareFootage * .85).toInt()
        } else {
            (rate * squareFootage).toInt()
        }

        println("\n\nCustomer name: $customerName")
        println("Customer phone: $customerPhone")
        println("Customer address: $customerAddress")
        println("Square Footage: $squareFootage")
        println("\n\n")
        println("Weekly Total:\t\t$$total")
    }
}